examples/cutting_stock_problem.rs

```diff
@@ -0,0 +1,179 @@
+use russcip::*;
```

Comment:
Could you also rename the example file to `cutting_stock`? I like the shorter name fits well the other `knapsack` example

Reply:
renamed